scripts/py/dcstat.py

```diff
@@ -152,17 +207,22 @@ def main():
             help='Verbosity level: 1) shows only DB names. 2) shows also failed tests names. 3) shows also errors log')
     parser.add_argument('-s', '--filter-species', type=str, nargs='+',
                         help='Filter results by species.')
+    parser.add_argument('-Q', '--quiet', action='store_true',
```

Comment:
good reason or typo in capitalized Q?

Reply:
No, that's intended. I use upper case for options that do not require an argument, and lower case for the argument ones. I'm not sure where that habit comes from actually. Maybe common GCC flags? I don't know.

Reply:
Well ok, first time I see this "rule" :-D no problem.